exporters/otlp/include/opentelemetry/exporters/otlp/otlp_exporter.h

```diff
@@ -50,9 +50,10 @@ class OtlpExporter final : public opentelemetry::sdk::trace::SpanExporter
    * Shut down the exporter.
    * @param timeout an optional timeout, the default timeout of 0 means that no
    * timeout is applied.
+   * @return return the status of this operation
    */
-  void Shutdown(
-      std::chrono::microseconds timeout = std::chrono::microseconds(0)) noexcept override{};
+  bool Shutdown(std::chrono::microseconds timeout = std::chrono::microseconds(0)) noexcept override{
+      return true};
```

Comment:
I think this is a syntax error, right?

```
return true;}
```

Reply:
Good catch, missed this as my local build did enable OTLP.